tasks/2.js

```diff
@@ -0,0 +1,24 @@
+// ES5
+var listOfNumbersES5 = [1, 2, 3, 4, 5, 6];
+
+function sumES5() {
```

Comment:
Nitpick, we expect this function to accept the array itself and operate on it instead of using hardcoded values. 😉